ci/Makefile

```diff
@@ -9,20 +9,15 @@ setup-conda-env:
 update-conda-env:
 	conda env update --name hail-ci -f environment.yml
 
-HAIL_CI_IMAGE_SHORT_NAME = hail-ci
```

Comment:
Not sure if you were using this.  Just seemed unnecessarily noisy and I changed it to match the structure of the other Makefiles.